package.json

```diff
@@ -43,7 +43,7 @@
     "babel-core": "^6.22.1",
     "babel-loader": "^6.2.10",
```

Comment:
Update all Webpack loaders to their latest version that explicitly support Webpack 2? (eslint-loader → 1.7.1 ; imports-loader → 0.7.1) Should we use the 7.0.0-beta.1 for babel-loader or live with the deprecation warning with 6.x?